CHANGELOG.md

```diff
@@ -3,6 +3,55 @@
 All notable changes to this project will be documented in this file.
 
 
+## Unreleased - as of Sprint 35 end 2016-01-25
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+35+Ending+Jan+25%2C+2016%22+label%3Aenhancement)
+
+- Providers
+  - OpenStack: cleanup SSL support, VM operations
+  - Google Cloud Engine: Inventory, Power Operations
+  - Microsoft SCVMM: Ability to delete VMs
+- REST API
+  - Added primary collection for Instances.
+  - Added terminate action for instances.
+  - Ability to filter string equality/inequality on virtual attributes.
+  - For SSUI, ability to retrieve user’s default language
+ and support for Dynamic Dialogs.
+- User Interface
+  - I18n: Marked translated strings directly in UI, Gettext support
+  - Containers
+    - Dashboard network trends
+    - Container environment variables table
+    - Search functionality for Container topology
+    - Dashboard no data cards
+    - Refresh option in Configuration dropdown
+- Automate
+  - SSUI: Support dialogs with dynamic fields
+  - Simulate UI support for state machine retries
+  - Service Models: Support where method, find_by, and find_by!
+- Ansible
+  - Modeling for Provider, Configuration Manager, and Configured Systems
+  - Provider connection logic
+  - Support refresh of Configured Systems
+  - Ansible Tower Client gem
```

Comment:
I'm thinking this should link to the gem...something like:

`- [ansible_tower_client](https://github.com/ManageIQ/ansible_tower_client) gem`

which looks like:
- [ansible_tower_client](https://github.com/ManageIQ/ansible_tower_client) gem